docs/make.jl

```diff
@@ -39,6 +39,7 @@ files = [
     # Topic: equations
     "Adding a new scalar conservation law" => "adding_new_scalar_equations.jl",
     "Adding a non-conservative equation" => "adding_nonconservative_equation.jl",
+    "Adding parabolic terms" => "adding_parabolic_terms.jl",
```

Comment:
I would remove the "adding" since we don't really add new functionality here, just combine existing equations. The other "adding" tutorials are about creating new equations for Trixi.jl.